### PR TITLE
Added issue templates

### PR DESCRIPTION
**Pull request by dlmarion**

### CODE REVIEW

**Comment by dlmarion** [comment]
@keith-turner @ctubbsii - I wanted to get your feedback on the post vote checklist specifically. I'm wondering if we can delete the `Update the website` section completely.

File: .github/ISSUE_TEMPLATE/post_vote_checklist.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+---
+
+- [ ] Label this issue with the GitHub [milestone](https://github.com/apache/accumulo-access/milestones) that corresponds to this release version
+- [Git](https://github.com/apache/accumulo-access) tasks
+  - [ ] Create a signed `rel/<version>` tag (and push it)
+  - [ ] Merge `<version>-rc<N>-next` branch into a maintenance branch (if maintenance is expected),
+        and then into the `main` branch (and push them)
+  - [ ] Remove `*-rc*` branches
+  - [ ] Update the Accumulo-Access dependency versions in the contrib projects
```
Comment (ctubbsii) [inline .github/ISSUE_TEMPLATE/post_vote_checklist.md:16]:
```suggestion
```
This part about contrib is still in flux, with some other PRs I'm working on.

File: .github/ISSUE_TEMPLATE/post_vote_checklist.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  - [ ] Remove old artifacts from the mirrors (**after** updating the website)
+- [Board reporting tool](https://reporter.apache.org/addrelease?accumulo)
+  - [ ] Add the date of release (the date the release artifacts were uploaded to SVN `UTC+0000`)
+- Verify published artifacts
+  - [ ] In [Maven Central](https://repo1.maven.org/maven2/org/apache/accumulo/accumulo-access/)
+  - [ ] In [ASF Downloads](https://downloads.apache.org/accumulo)
+  - [ ] In [several mirrors or CDN](https://www.apache.org/dyn/closer.lua/accumulo)
+- Update the [website](https://accumulo.apache.org/)
+  - [ ] Release notes
+  - [ ] Add `LTM: true` to the release notes front-matter for LTM releases
```
Comment (ctubbsii) [inline .github/ISSUE_TEMPLATE/post_vote_checklist.md:31]:
```suggestion
```

LTM doesn't really apply to this library, I think. We'll still want to do release notes, though, probably.

File: .github/ISSUE_TEMPLATE/post_vote_checklist.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+  - [ ] Navigation
+  - [ ] Downloads page
+  - [ ] If LTM release, update previous LTM release entry on downloads page and release notes with an EOL date 1 year from the current release date
+  - [ ] DOAP file
```
Comment (ctubbsii) [inline .github/ISSUE_TEMPLATE/post_vote_checklist.md:35]: We'll create a new DOAP file just for this library, separate from the main accumulo one. We have precedent for this, with the maven plugin. See the doap directory for details. We currently don't have docs or navigation about this library, or anything on the download page. We'll probably want to add that at some point after the first release.

File: .github/ISSUE_TEMPLATE/post_vote_checklist.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+  - [ ] Add manual/examples/javadoc (build javadoc from tag with `mvn clean package -DskipTests javadoc:aggregate -Paggregate-javadocs`)
+  - [ ] Add updated properties documentation (see [the website README](https://github.com/apache/accumulo-website#updating-property-documentation-for-releases))
+  - [ ] Jekyll config
+  - [ ] Grep for, and update any links to previous version to now point to the new version
+  - [ ] Update any older release notes front-matter to indicate they are either `archived: true` or `archived_critical: true`
```
Comment (ctubbsii) [inline .github/ISSUE_TEMPLATE/post_vote_checklist.md:40]: These parts don't really apply to this library.

File: .github/ISSUE_TEMPLATE/post_vote_checklist.md [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+- Announcement email
+  - [ ] Prepare and get review on dev list (see examples [from previous announcement messages](https://lists.apache.org/list.html?announce@apache.org:gte=1d:accumulo))
+  - [ ] Send to announce@apache.org and user@accumulo.apache.org (use plain text mode only; html email will be rejected)
+- GitHub wrap-up
+  - [ ] Close this issue
+  - [ ] Create a new GitHub milestone for the next version (if necessary) and move any open issues not completed in this release to that project
+  - [ ] Close the project that corresponds to this release
```
Comment (ctubbsii) [inline .github/ISSUE_TEMPLATE/post_vote_checklist.md:47]:
```suggestion
  - [ ] Close the milestone that corresponds to this release
```

**Review by ctubbsii — COMMENTED** [review]
I'm not really sure it's necessary to create these templates for the smaller project. People rarely use them for the main project, and I doubt there's going to be that much activity on this library once we've got a version released that's solid and stable. We could probably just skip doing any of this.

**Comment by dlmarion** [comment]
>  We could probably just skip doing any of this.

That's fair, but I think at a minimum we should to the post-vote checklist as it's not immediately clear to me, and likely anyone that rarely performs releases, what needs to be done. I'll remove the other files and update the post-vote checklist with your suggestions.

**Pull request merged**